_env/nginx.conf

```diff
@@ -82,6 +82,10 @@ http {
       proxy_pass https://patience.studentrobotics.org/docs/python/;
     }
 
+    location /rules/ {
+      proxy_pass https://patience.studentrobotics.org/docs/rules/;
```

Comment:
I'm not sure we can spell this as we want using a `location` block. I think we want a `rewrite` tagged as a redirect (there's already a bunch of them lower down).

I think we want something like:
``` nginx
    rewrite ^/rules                /docs/rules permanent;
```